packages/schematics/angular/ng-new/schema.json

```diff
@@ -95,19 +96,33 @@
     "routing": {
       "type": "boolean",
       "description": "Generates a routing module.",
-      "default": false
+      "default": false,
+      "x-prompt": "Will the project use Angular routing?"
```

Comment:
What tense & tone should be used for prompts?

It feels like this question should be more about what WE will do, like 'Would you like to generate a routing module?' or 'Should we add routing to the project?'

Reply:
Updated.